### PR TITLE
adjust padding for QR Images

### LINKED ISSUE #3504: removing padding surrounding profile photo in the centre of QR to match designs.
# Problem

The current implementation produces a QR code on the media server that looks like this : 

Simulator Screenshot - iPhone 14 Pro - 2023-05-18 at 11 58 56

The designs were updated to remove the white padding surrounding the profile photo : 

Screenshot 2023-05-18 at 12 04 59 PM

### PR DESCRIPTION
**Pull request by siddarthkay**
I've adjusted the padding to make the QR code more in line with the designs.
Here is how it looks now

Simulator Screenshot - iPhone 14 Pro - 2023-05-19 at 18 52 57

Closes #3504

### CODE REVIEW

**Review by qfrank — APPROVED** [review]

**Comment by qfrank** [comment]
Don't forget bump version 🙂 @siddarthkay

**Review by Samyoul — APPROVED** [review]

**Review by ilmotta — APPROVED** [review]

File: images/manipulation.go [hunk 1]
```diff
@@ -155,14 +155,42 @@ func EncodePNG(img *image.RGBA) ([]byte, error) {
 	return resultImg.Bytes(), nil
 }
 
-func CreateCircle(img image.Image) *image.RGBA {
+// helper function to calculate the minimum of two integers
```
Comment (ilmotta) [inline images/manipulation.go]: The convention I think is to start the documentation with the name of the function.

Comment (siddarthkay) [reply images/manipulation.go]: Got rid of it, I thought about it a bit more and it seems like that operation was not necessary.

**Comment by siddarthkay** [comment]
so unit tests have been consistently timing out because they exceed the 20 minute threshold.

According to last logs the tests begin at : `2:06:46 AM on May 24, 2023`
and then they end or are max timed out at : `2:27:06 AM on May 24, 2023`

logs download link : https://ci.status.im/blue/rest/organizations/jenkins/pipelines/status-go/pipelines/prs/pipelines/tests/branches/PR-3510/runs/11/log/?start=0&download=true

**Comment by siddarthkay** [comment]
Here is a rough breakdown of the 20 minutes : 

- 3 minutes for this activity over here 
```log
[2023-05-24T02:06:46.677Z] ok  	github.com/status-im/status-go/account/generator	(cached)
[2023-05-24T02:06:46.677Z] ?   	github.com/status-im/status-go/account/json	[no test files]
[2023-05-24T02:09:41.911Z] === RUN   TestParseAppType
[2023-05-24T02:09:41.911Z] --- PASS: TestParseAppType (0.00s)
[2023-05-24T02:09:41.911Z] === RUN   TestBackendStartNodeConcurrently
```
- 18 minutes over here 🫣
```log
2023-05-24T02:09:41.918Z] --- PASS: TestProfilingMem (0.00s)
[2023-05-24T02:09:41.918Z] PASS
[2023-05-24T02:09:41.918Z] ok  	github.com/status-im/status-go/profiling	(cached)
[2023-05-24T02:27:00.615Z] === RUN   TestDeleteActivityCenterNotificationsWhenEmpty
[2023-05-24T02:27:00.615Z] --- PASS: TestDeleteActivityCenterNotificationsWhenEmpty (18.74s)
[2023-05-24T02:27:00.615Z] === RUN   TestDeleteActivityCenterNotificationsWithMultipleIds
[2023-05-24T02:27:00.615Z] --- PASS: TestDeleteActivityCenterNotificationsWithMultipleIds (19.16s)
```

**Comment by siddarthkay** [comment]
One more interesting thing is a bunch of these : 
- `[2023-05-24T02:27:00.627Z] 2023-05-24T02:11:23.125Z	ERROR	node/node.go:195	Failed to persisten epoch	{"mvds": {"error": "sql: database is closed"}}`
approximately 1,400 of these :)

- `[2023-05-24T02:27:00.637Z] 2023-05-24T02:11:27.229Z	 DEBUG    common/message_sender.go:633	failed to handle an encryption message {"site": "handleMessages", "hash": xxxx}`
approximately 200 of these

- `[2023-05-24T02:27:00.638Z] 2023-05-24T02:11:27.408Z	ERROR	common/message_sender.go:683	failed to handle application metadata layer message	{"site": "handleMessages", "hash":xxxx}`
approximately 13 of these

**Comment by siddarthkay** [comment]
So to get some readable logs i wrote a bash script to get me all tests that Passed and then sorted them in descending order.

```bash
#!/bin/bash

input_log_file=$1

awk '/--- PASS/ { print }' "$input_log_file" | \
awk -F'[()]' '{ print $2, $0 }' | sort -rn | cut -d' ' -f2- > "simplified_sorted_logs.txt"
```


Listing tests here that take more than 40 seconds :

```log
[2023-05-24T02:27:06.311Z] --- PASS: TestEncryptionServiceMultiDeviceSuite (445.05s)
[2023-05-24T02:27:06.310Z] --- PASS: TestManagerSuite (223.26s)
[2023-05-24T02:27:03.490Z] --- PASS: TestMessengerProfilePictureHandlerSuite/TestE2eSendingReceivingProfilePicture (194.05s)
[2023-05-24T02:27:06.376Z] --- PASS: TestSQLLitePersistenceTestSuite (190.24s)
[2023-05-24T02:27:06.390Z] --- PASS: TestServerSuite (160.78s)
[2023-05-24T02:27:06.311Z] --- PASS: TestEncryptionServiceMultiDeviceSuite/TestMaxDevices (146.05s)
[2023-05-24T02:27:06.310Z] --- PASS: TestPersistenceSuite (132.09s)
[2023-05-24T02:27:06.311Z] --- PASS: TestEncryptionServiceMultiDeviceSuite/TestMaxDevicesRefreshedBundle (131.34s)
[2023-05-24T02:27:06.376Z] --- PASS: TestClientSuite (120.90s)
[2023-05-24T02:27:06.376Z] --- PASS: TestSQLitePersistenceSuite (114.64s)
[2023-05-24T02:27:06.376Z] --- PASS: TestSQLitePersistenceSuite (96.52s)
[2023-05-24T02:27:06.306Z] --- PASS: TestMessageSenderSuite (96.23s)
[2023-05-24T02:27:06.311Z] --- PASS: TestEncryptionServiceMultiDeviceSuite/TestProcessPublicBundle (86.00s)
[2023-05-24T02:27:00.795Z] --- PASS: TestMessengerCommunitiesSuite (83.83s)
[2023-05-24T02:27:06.311Z] --- PASS: TestEncryptionServiceMultiDeviceSuite/TestProcessPublicBundleOutOfOrder (81.67s)
[2023-05-24T02:27:06.402Z] --- PASS: TestSyncDeviceSuite (72.36s)
[2023-05-24T02:27:06.405Z] --- PASS: TestShhExtSuite (62.60s)
[2023-05-24T02:27:04.625Z] --- PASS: TestMessengerSuite (50.87s)
[2023-05-24T02:27:06.376Z] --- PASS: TestServiceTestSuite (48.85s)
[2023-05-24T02:27:06.306Z] --- PASS: TestMessageSenderSuite/TestHandleDecodedMessagesDatasyncEncrypted (48.75s)
[2023-05-24T02:27:06.397Z] --- PASS: TestPersistenceSuite (47.84s)
[2023-05-24T02:27:01.249Z] --- PASS: TestMessengerContactRequestSuite (45.02s)
[2023-05-24T02:27:04.229Z] --- PASS: TestMessengerSyncSettings (42.73s)
```
So they easily go beyond 20 minutes, we either speed some of the long running ones
OR
we increase the max timeout value

**Comment by ilmotta** [comment]
> So they easily go beyond 20 minutes, we either speed some of the long running ones OR we increase the max timeout value

If I'm not mistaken, recently we increased the timeout to 20min, which is already painfully slow for continuous integration purposes. One approach would be to categorize tests by their speed, reliability, etc and see if we can start to optimize the test suite based on the ones that are slower/flaky and so on.

**Comment by siddarthkay** [comment]
Hmm Okay, any test that takes more than 100 seconds or so should be looked at.. I think this deserves its own github issue, I'll create one and tag folks who made these tests for their input. The folks that wrote these tests would have better context on how to optimise them :)

**Comment by ilmotta** [comment]
> Hmm Okay, any test that takes more than 100 seconds or so should be looked at.. I think this deserves its own github issue, I'll create one and tag folks who made these tests for their input. The folks that wrote these tests would have better context on how to optimise them :)

@siddarthkay, I think both approaches (and others) are simultaneously valid. My comment was more as an addendum to the problem you described. If we get blocked by the CI, raising the timeout might be the only palliative solution.

**Comment by siddarthkay** [comment]
After a million force-push later tests are green ✅

**Comment by qfrank** [comment]
> After a million force-push later tests are green ✅

you don't have to do force-push to trigger rerun test, you can just click `Details`(the failed one) and rerun test by clicking `Restart Unit Tests` @siddarthkay

**Comment by siddarthkay** [comment]
@qfrank : wow thanks! I didn't know we could do that, I think @jakubgs once said not to re run any pipeline stages in isolation.

**Pull request merged**